scripts/install-wasm-bindgen.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -eu
+
+cd $(dirname "$0")/..
+
+# Install binstall
+curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
+
+# Install wasm-bindgen-cli.
+if [ "$(wasm-bindgen --version | cut -d' ' -f2)" != "0.2.90" ]; then
+  echo "Building wasm-bindgen..."
+  cargo binstall wasm-bindgen-cli@0.2.90 --force --no-confirm
```

Comment:
Why is this relatively complicated binstall way needed? Wouldn't a simple `cargo install wasm-bindgen-cli` do it as well?

Reply:
Hey great question, 

`cargo install wasm-bindgen-cli` will always recompile the code from source, so if you look at the CI times for [dd8dfc0](https://github.com/ethereumjs/verkle-cryptography-wasm/pull/7/commits/dd8dfc0e35645c81246674a94b4ed4298dde3f5d) its about 2-3 minutes due to it recompiling wasm-bindgen-cli from source. 

cargo binstall is a tool that will go into the github repo for wasm-bindgen-cli and check the release page for the binary with version 0.2.90, so it shaves off about 1-2 minutes off of the ci each time its ran, because its just downloading the binary instead of recompiling

Reply:
👍